src/helpers/Sentry.ts

```diff
@@ -0,0 +1,32 @@
+import * as Sentry from '@sentry/browser';
+import createSentryMiddleware from 'redux-sentry-middleware';
+import config from '../config';
+
+export default {
+
+  initializeWithMiddleware() {
+    Sentry.init({
+      dist: config.CODE_VERSION,
```

Comment:
release_id is more descriptive, that captures code version and books that are built